README.md

```diff
@@ -47,25 +47,39 @@ SCM][jenkins-doc-pipelineFromSCM]. The location of the `Jenkinsfile` may
 differ:
 
 1. The `Jenkinsfile` can be copied into the source code repository of the
-   application. In that case, no additional pipeline job parameters are required
-   to retrieve the application source code repository.
+   application. The `Jenkinsfile` is referenced in the pipeline job
+   configuration.
 
 2. The `Jenkinsfile` can be kept in its own repository separate from the
    application. In case you want to apply the `Jenkinsfile` to multiple
    applications, this approach is beneficial since you only need to maintain the
    `Jenkinsfile` in one place. You can consume the `Jenkinsfile` e.g. directly
-   from this repository on GitHub.com or from your own fork of it. You must
-   define two mandatory Jenkins pipeline job parameters to specify the location
-   of your application:
-
-    * `GIT_URL`: The URL to the Git repository that the application resides in.
-    * `GIT_BRANCH`: The branch of the Git repository that should be checked out
-      for the build.
-
-Example configuration for consumption of a `Jenkinsfile` directly out of this
-repository on GitHub.com: 
-
-![Library Setup](./documentation/docs/images/setupInJenkins.png)
+   from this repository on GitHub.com or from your own fork of it.
+   In this case there is a stub `Jenkinsfile` required in the source repository.
+   The stub `Jenkinsfile` needs to be configured in the pipeline job definition.
+   The stub `Jenkinsfile` needs to contain the `pipelineExecute` execute step as
+   it is contained in the piper library.
+   The stub `Jenkinsfile` looks like
+```groovy
+#!groovy
+
+@Library('piper-library-os') _
+
+// The coordinates of the central pipeline script
+def REPO = '<REPO>'
```

Comment:
I thought about providing the coordinates as job parameters. I doubt it is a good idea. With that each job execution can happen with a different central pipeline. It is more obvious to have it hard coded in the pipeline. I also did not provided over-writable defaults since this makes the example more difficult to understand.

Reply:
Good point. But we could put it into the config file, right? Then, we still hit the target that the Jenkinsfile itself is consumable out of the box.

Reply:
We can use an additional config file. I'm not strongly against such an approach. But for me it seems to be some kind of over-optimization. We provide an example and the tranfer we expect from the user is to replace some strings which act obviously as PLACE_HOLDERS. It guess that is not too much ... What is the difference of adapting a config file or a stub pipeline itself? It is a change in the sources anyway. Benefit would be: pipeline runs "out of the box", but on the price we have a more complex example. For didactical reasons I think the example should be as straight forward as possible. But again: I'm not _strongly_ against a config file appraoch.